resources/web/wwi/protoVisualizer/Parameter.js

```diff
@@ -27,10 +27,36 @@ export default class Parameter extends Field {
       throw new Error(`Type mismatch, setting ${stringifyType(newValue.type())} to ${stringifyType(this.type)} parameter.`);
 
     if (this.restrictions.length > 0) {
-      for (const item of this.restrictions) {
-        if ((newValue instanceof SFNode && newValue.value === null) || item.equals(newValue)) {
+      if (newValue instanceof MFNode) {
+        if (newValue.value === null) {
```

Comment:
I think this conditional (newValue.value === null) should be checked only for SFNodes, MFNodes can't be `null`. At best they can be `[]`, but that's taken care of already